napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -162,7 +162,7 @@ def __init__(self, layer):
         )
 
         self.panzoom_button = QtModeRadioButton(
-            layer,
+            self.layer,
```

Comment:
curious why all these? pulled this PR and it doesn't seem to make a difference for the tests.

Reply:
try to play with reference, but obsolete